kuka_rox_hardware_interface/src/robot_manager_node.cpp

```diff
@@ -181,6 +181,14 @@ void RobotManagerNode::ObserveControl()
 
   while (reader->Read(&response)) {
     switch (static_cast<int>(response.event())) {
+      case kuka::ecs::v1::CommandEvent::COMMAND_MODE_SWITCH:
+        {
+          std::lock_guard<std::mutex> lk(control_mode_cv_m_);
+          control_mode_change_finished_ = true;
+        }
+        RCLCPP_INFO(get_logger(), "Command mode switched");
```

Comment:
Switched or is in progress? I think both are valid, because the switch already happened, but when we receive this msg, the new session might not be active. Just be consistent with the other message (since it'd be weird to see a print like "Command mode switched" before the "Command mode switch in progress message")